packages/controller-utils/CHANGELOG.md

```diff
@@ -7,6 +7,12 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [11.5.0]
```

Comment:
I was thinking it should even be major, since it could be considered a breaking change that clients start getting new values.  But it's not a very big change, so I left it minor.

Reply:
Gotcha. Just to ensure we are on the same page, when we refer to "breaking" changes it doesn't mean that MetaMask users will be surprised, it means that a developer upgrades to the new version, the client will cease to build (or, worse, the client will still build but existing functionality will be invisibly broken somehow). If everything still works as designed without needing to update dependencies or code, then it's not breaking. Clients are free to signify and announce changes to end users however they think best — but here the target audience is the developer.